i18n-helpers/src/xgettext.rs

```diff
@@ -39,14 +39,15 @@ fn strip_link(text: &str) -> String {
     without_link
 }
 
-fn add_message(catalog: &mut Catalog, msgid: &str, source: &str) {
+fn add_message(catalog: &mut Catalog, msgid: &str, source: &str, comment: &str) {
     let sources = match catalog.find_message(None, msgid, None) {
         Some(msg) => format!("{}\n{}", msg.source(), source),
         None => String::from(source),
     };
     let message = Message::build_singular()
         .with_source(sources)
         .with_msgid(String::from(msgid))
+        .with_comments(comment.into())
```

Comment:
Let us make this consistent with the line above:

```suggestion
        .with_comments(String::from(comment))
```

Using `.into()` or `T::from()` is a bit of a style choice: I personally like the `T::from()` form since the `T` type is explicit. Others prefer the `.into()` calls instead. Both are valid, we should just be consistent.